rasa/core/broker.py

```diff
@@ -106,9 +160,11 @@ def _open_connection(self):
             # host seems to be just the host, so we use our parameters
             parameters = pika.ConnectionParameters(
                 self.host,
+                port=self.port,
                 credentials=self.credentials,
                 connection_attempts=20,
                 retry_delay=5,
+                ssl_options=create_rabbitmq_ssl_options(self.host),
```

Comment:
we've tested this still works if no certs are passed?

Reply:
yes 👍